shinhye/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const dotenv = require("dotenv");
+const morgan = require('morgan');
+
+dotenv.config();
+
+app.use(cors());
+app.use(morgan('combined'));
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
```

Comment:
typeorm의 공식문서를 확인해주세요! (구두 전달 완료)

```suggestion
myDataSource.initialize()
  .then(() => {
    console.log("Data Source has been initialized!");
  })
  .catch((err) => {
    console.error(' 에러메시지 ');
  });
```

Reply:
확인 후 반영했습니다 감사합니다!